Mathlib/MeasureTheory/Function/LpSeminorm.lean

```diff
@@ -105,19 +109,25 @@ theorem snorm_exponent_top {f : α → F} : snorm f ∞ μ = snormEssSup f μ :=
 #align measure_theory.snorm_exponent_top MeasureTheory.snorm_exponent_top
 
 /-- The property that `f:α→E` is ae strongly measurable and `(∫ ‖f a‖^p ∂μ)^(1/p)` is finite
-if `p < ∞`, or `ess_sup f < ∞` if `p = ∞`. -/
-def Memℒp {α} {m : MeasurableSpace α} (f : α → E) (p : ℝ≥0∞)
-    (μ : Measure α := by exact MeasureTheory.MeasureSpace.volume) : Prop :=
+if `p < ∞`, or `essSup f < ∞` if `p = ∞`. -/
+def Memℒp {α} {_ : MeasurableSpace α} (f : α → E) (p : ℝ≥0∞)
+    (μ : Measure α := by volume_tac) : Prop :=
   AEStronglyMeasurable f μ ∧ snorm f p μ < ∞
 #align measure_theory.mem_ℒp MeasureTheory.Memℒp
 
-theorem Memℒp.aEStronglyMeasurable {f : α → E} {p : ℝ≥0∞} (h : Memℒp f p μ) :
+theorem memℒp_eq {α} {_ : MeasurableSpace α} (f : α → E) (p : ℝ≥0∞) (μ : Measure α) :
+    Memℒp f p μ = (AEStronglyMeasurable f μ ∧ snorm f p μ < ∞) :=
+  rfl
+
+attribute [eqns memℒp_eq] Memℒp
```

Comment:
Why do we want this lemma? If we are going to have it, I think it would be to state it using an `Iff`

Reply:
I've added the porting note and changed `Eq` with `Iff`.